packages/excalidraw/data/reconcile.ts

```diff
@@ -20,7 +20,7 @@ const shouldDiscardRemoteElement = (
     // local element is being edited
     (local.id === localAppState.editingElement?.id ||
       local.id === localAppState.resizingElement?.id ||
-      local.id === localAppState.draggingElement?.id || // TODO: Is this still valid? As draggingElement is selection element, which is never part of the elements array
+      local.id === localAppState.newElement?.id || // TODO: Is this still valid? As newElement is selection element, which is never part of the elements array
```

Comment:
@Mrazator should we also prefer local element if it's being dragged? Not that we can detect it efficiently right now as we don't track dragged elements

Reply:
This part is potentially incorrect. If we want to preserve local element that's being dragged, we could update the logic. Otherwise, we should just remove the changed line of code.

Reply:
> should we also prefer local element if it's being dragged

probably, though now thinking that these conditions could momentarily lead to an inconsistent reconciliation result on different clients; though such cases will immediately be fixed with the next update, so it is likely not necessarily a big issue

Reply:
yeah, though it's incorrect in production as well. Now at least we prefer elements that are being currently created, though it likely doesn't matter and it's unlikely those will be updated on remote (unless there's a bug in sync).

If we want to prefer local dragged elements, we'd likely need to introduce a new state for all dragged elements (map preferably, for perf).